docs/npwd/start/Install.md

```diff
@@ -0,0 +1,102 @@
+---
+id: installation
+title: Install and Setup
+sidebar_label: Installation
+---
+
+### Third-Party Resource Dependencies
+If you wish for NPWD to have full functionality. The following resources are required to be
+installed and started before NPWD.
+
+1. [screenshot-basic](https://github.com/project-error/screenshot-basic) - **Make sure to use the exact version, as linked.**
+2. [pma-voice](https://github.com/AvarianKnight/pma-voice) - Support for other VOIP resources is planned (`toko-voip`) etc
+
+:::note
+NPWD has standardized pma-voice as our default voice system for the phone. We encourage everyone to do such.
+:::
+
+## Installation and Setup
+
+### Downloading Release
+1. Download the latest release of NPWD from the [releases page](https://github.com/project-error/npwd/releases)
+2. Unzip and place the `npwd` folder in your FXServer's `resources` directory.
+
+### Setting up with a Framework
+If you are using a popular framework such as ESX or QBCore, you can easily integrate NPWD using the corresponding
+premade compatbility resources, these should automatically configure NPWD to work with your framework.
+
+* [esx-npwd](https://github.com/overextended/esx-npwd)
```

Comment:
A disclaimer here is needed as lindens ESX has this fully integrated. So they wouldn’t use this.